common/enums/defaults.go

```diff
@@ -45,3 +45,9 @@ func SetDefaultTaskListKind(f *enumspb.TaskListKind) {
 		*f = enumspb.TASK_LIST_KIND_NORMAL
 	}
 }
+
+func SetDefaultParentClosePolicy(f *enumspb.ParentClosePolicy) {
+	if *f == enumspb.PARENT_CLOSE_POLICY_UNSPECIFIED {
+		*f = enumspb.PARENT_CLOSE_POLICY_ABANDON
```

Comment:
The default should be TERMINATE. 
How do I determine default looking at the proto definition?